am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestDataFactory.java

```diff
@@ -28,6 +29,23 @@ public static ExplicitAccessRecord createRecord(String resourceId,
             .build();
     }
 
+    public static ExplicitAccessRecord createRecord(String resourceId,
+                                                    String accessorId,
+                                                    Set<Permission> explicitPermissions,
```

Comment:
Can you swap attribute and permissions. It feels more natural to me to have attribute first. Happy to talk if you disagree.

Reply:
👍